HACKS.md

```diff
@@ -14,22 +14,6 @@ Tell us when we can remove this hack.
 
 > 👀 When adding a new item, see comment on top of file for template.
 
-## Jest's JSDom pinned to v13
```

Comment:
🤩 Nice to see a `HACKS` document actually shrink in size!

Thanks for cleaning this up and getting us over this hump.